tests/std/tests/P1502R1_standard_library_header_units/test.cpp

```diff
@@ -92,8 +92,8 @@ import <vector>;
 import <version>;
 // clang-format on
 
-#include <assert.h>
-#include <stdio.h>
+#include <cassert>
+#include <cstdio>

```

Comment:
I'm somewhat concerned about this change, as the Header Units test is special. While the test is continuing to pass, now this is classic-`#include`-ing `<yvals_core.h>` indirectly, after having imported it (indirectly) as a deduplicated header unit. Having the `import`s be the only source of STL machinery is cleaner for the purposes of this test. I would prefer to undo this change and comment the `<meow.h>` lines as being special.